lib/validators/liveValidator.ts

```diff
@@ -9,7 +9,7 @@ import * as msRest from "ms-rest"
 import * as os from "os"
 import * as path from "path"
 import { ParsedUrlQuery } from "querystring"
-import * as url from "url"
+import * as URL from "url"
```

Comment:
It should be `url`. Even is it's an abbreviation.

Reply:
eh I changed it since I was getting some shadowed variable tslint errors